modules/ROOT/pages/autocorrect.adoc

```diff
@@ -0,0 +1,94 @@
+= Spelling Autocorrect plugin
+:navtitle: Spelling Autocorrect
+:description: Autocorrect common spelling errors.
+:keywords: autocorrect, capitalization, misprint, spell, spelling, spellcheck, typo, typographical error
+:pluginname: Spelling Autocorrect
+:plugincode: autocorrect
+:pluginminimumplan: tierthree
+
+
+include::partial$misc/admon-premium-plugin.adoc[]
+
+include::partial$misc/admon-requires-6.2v.adoc[]
+
+The {pluginname} plugin allows {productname} to autocorrect a specified set of spelling errors, common typos, and capitalization errors, as they are being typed.
+
+== Interactive Example
+
+liveDemo::autocorrect[]
+
+include::partial$misc/purchase-premium-plugins.adoc[]
+
+== Basic setup
+
+[source,js]
+----
+tinymce.init({
+  selector: 'textarea',  // change this value according to your HTML
+  plugins: 'autocorrect'
+});
+----
+
+[[usage-notes]]
+== Usage notes
+
+The {pluginname} plugin currently has the following limitations:
+
+. English is the only supported language.
+
+. Capitalization and autocorrect can both be turned off, but specific autocorrections cannot.
++
+{pluginname} can be entirely turned off, but there is, currently, no way of instructing the plugin to ignore a particular string that it considers a mis-spelling.
++
+For example, if *cna* (which the {pluginname} plugin treats as a mis-typing of *can*) is a desired string in a particular use-case, turning {pluginname} off is the currently recommended setting.
+
+. End-users cannot, currently, turn Autocorrect on or off on hand-held devices.
+
+. Spelling Autocorrect is case-sensitive.
++
+As a consequence, mis-spelled words are not autocorrected if they do not exactly match the plugin’s built-in correction set.
++
+For example, {pluginname} autocorrects *teh* to *the*, but does not, currently, autocorrect *Teh* to *The*.
+
+[[known-issues]]
+== Known issues
+
+The {pluginname} plugin has the following known issues:
+
+. Autocorrect over-rides auto-capitalization.
++
+As a consequence of {pluginname} being case-sensitive, if
++
+  * both `autocorrect` and `capitalization` are active; and
+  * a word from the specified spelling error set is mis-typed at the beginning of a sentence;
++
+the spelling is corrected but the initial letter is not capitalized.
+
+. Auto-capitalization does not work on text immediately following a soft return in non-empty {productname} documents.
++
+If
++
+  * a {productname} document contains any data other than a soft return character (Unicode U+2028, commonly entered by typing *Shift+Return*); and
+  * the character immediately preceding an entered string is a soft return character
++
+Auto-capitalization will not operate on the entered string.
```

Comment:
```suggestion
Auto-capitalization will not operate on the entered string if
+
  * a {productname} document contains any data other than a soft return character (Unicode U+2028, commonly entered by typing *Shift+Return*); and
  * the character immediately preceding an entered string is a soft return character.
+
```
I'm not sure what your preference is, just putting `If` on a single line followed by a list and then a final line looks funny to me.